apps/server/src/app/export/msword.ts

```diff
@@ -40,6 +58,225 @@ export class MSWordExporter {
+                this.body.push(new Paragraph({ text: `Max: ${element.max}` }));
+                this.body.push(new Paragraph({ text: `Must support: ${element.mustSupport}` }));
+                this.body.push(new Paragraph({ text: `Binding: ${element.binding}` }));
+                this.body.push(new Paragraph({ text: `Short: ${element.short}` }));
+                this.body.push(new Paragraph({ text: `Label: ${element.label}` }));
+
+              }
+            );
+          };
+
+
+
+            keys.forEach((key) => {
+            //if(key === 'name' || 'url' || 'description'){
+            const table = new Table({
+              rows: [
+                new TableRow({
+                  children: [
+                    new TableCell({
+                      children: [new Paragraph(key)],
+                      verticalAlign: VerticalAlign.CENTER,
+                    }),
+                    new TableCell({
+                      children: [new Paragraph(entry.resource[key])],
+                      verticalAlign: VerticalAlign.CENTER,
+                    }),
+                  ]
+                })
+              ]
+            });
+            this.body.push(table);
+          //}
+          });
+
+          this.body.push(new Paragraph({
+            text: ""
+          }));
+          }
+        );
+
+      bundle.entry
+        .filter(entry => entry.resource && (entry.resource.resourceType === 'ValueSet'))
+        .forEach((entry) => {
+            const extensionUrlKeys = Object.keys(Globals.extensionUrls);
+            const extensionUrls = extensionUrlKeys.map((key) => Globals.extensionUrls[key]);
+            const keys = Object.keys(entry.resource);
+
+            keys.forEach((key) => {
+              //if(key === 'name' || 'url' || 'description'){
+              const table = new Table({
+                rows: [
+                  new TableRow({
+                    children: [
+                      new TableCell({
+                        children: [new Paragraph(key)],
+                        verticalAlign: VerticalAlign.CENTER,
+                      }),
+                      new TableCell({
+                        children: [new Paragraph(entry.resource[key])],
+                        verticalAlign: VerticalAlign.CENTER,
+                      }),
+                    ]
+                  })
+                ]
+              });
+              this.body.push(table);
+              //}
+            });
+
+            this.body.push(new Paragraph({
+              text: ""
+            }));
+          }
+        );
+
+      bundle.entry
+        .filter(entry => entry.resource && (entry.resource.resourceType === 'CodeSystem'))
+        .forEach((entry) => {
+            const extensionUrlKeys = Object.keys(Globals.extensionUrls);
+            const extensionUrls = extensionUrlKeys.map((key) => Globals.extensionUrls[key]);
+            const keys = Object.keys(entry.resource);
+
+            keys.forEach((key) => {
```

Comment:
I'd rather output a select number of properties, rather than just looping through all the keys...
* name
* title
* purpose
* description
* filter
  * filter.code
  * filter.description
* property
  * property.code
  * property.description
* total number of concept